### PR TITLE
Allow search-service to handle new hits type "favorited_search" 

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-6341

### CODE REVIEW

File: src/models/hit-types/favorited_search-hit.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  @Memoize() get query(): typeof Metadata.prototype.query {
+    return this.rawMetadata?.fields?.query
+      ? new StringField(this.rawMetadata.fields.query)
+      : undefined;
+  }
+
+  /**
+   * Optional.
+   */
+  @Memoize() get date_favorited(): typeof Metadata.prototype.date_favorited {
```
Comment (latonv) [inline src/models/hit-types/favorited_search-hit.ts:55]: 👍

File: package.json [hunk 2]
```diff
@@ -75,7 +75,7 @@
     ]
   },
   "dependencies": {
-    "@internetarchive/field-parsers": "^0.1.3",
+    "@internetarchive/field-parsers": "0.1.3-alpha1",
```
Comment (latonv) [inline package.json:78]: Q: What was updated in the field parsers?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internetarchive/search-service",
-  "version": "1.2.0",
+  "version": "1.2.0-alpha3",
```
Comment (latonv) [inline package.json:3]: Reminder to undo alpha version before merging this.

**Review by latonv — CHANGES_REQUESTED** [review]
Thanks for the hit type addition!

Noted some testing changes that block merge. Otherwise LGTM aside from a couple of minor Qs/suggestions.

File: package.json (hunk 2 above)
Comment (nsharma123) [reply package.json:78]: adding new mediatype type `search`.

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,5765 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
[the remark this answers is not in the document]
Comment (nsharma123) [reply yarn.lock:1]: yes, its mistake. reverting.

**Pull request merged**